values/prometheus-operator/prometheus-operator.gotmpl

```diff
@@ -266,13 +269,62 @@ grafana:
   additionalDataSources:
     - name: Loki
       editable: false
+      uid: loki
       type: loki
       access: proxy
       url: http://loki-query-frontend-headless.monitoring:3101
       basicAuth: true
       basicAuthUser: otomi-admin
       secureJsonData:
         basicAuthPassword: {{ $v.apps.loki.adminPassword }}
+    {{- if $c.tempo.enabled }}
+      jsonData:
+        derivedFields:
+          - datasourceName: Tempo
+            matcherRegex: "traceID=00-([^\\-]+)-"
+            name: TraceID
+            url: "$${__value.raw}"
+            datasourceUid: tempo
+    - name: Tempo
```

Comment:
It looks like there is a big overlap with the configuration that you added to the `helmfile.d/helmfile-60.teams.yaml` file. Why is it necessary?

Reply:
Because there is more than 1 grafana instance (platform grafana and team's grafana). Both have separate configs